source/js/components/TomatoSlider.js

```diff
@@ -14,11 +18,24 @@ class TomatoSlider extends HTMLElement {
    */
   constructor() {
     super();
-    this.appendChild(document.querySelector('#tomato-slider-template').content.cloneNode(true));
+
+    this.svgUrls = svgIcons[backend.get('Icon')].urls;
+    this.svgClasses = svgIcons[backend.get('Icon')].classes;
+    // Create template and append to tomato-slider (need to maintain input child)
+    const template = document.createElement('template');
+    template.innerHTML = `
+      <div class="d-flex h-100 justify-content-between align-items-center slider-tomato-container">
```

Comment:
So, I believe the 'h-100' class is what causes the rows to load drastically in height before all the icons are re rendered. If we could adjust the styling so the row heights remain consistent during icons loading and is not noticeable to user.

Reply:
Fix done by Mark